chalk-ir/src/debug.rs

```diff
@@ -21,6 +21,13 @@ impl<I: Interner> Debug for AssocTypeId<I> {
     }
 }
 
+impl<I: Interner> Debug for Ty<I> {
+    fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), Error> {
+        I::debug_ty(self, fmt).unwrap_or_else(|| unimplemented!())
```

Comment:
dont know what the default here should be

Reply:
yeah that's a tricky one